pkg/controller/admissionchecks/multikueue/jobset_adapter_test.go

```diff
@@ -223,12 +223,16 @@ func TestWlReconcileJobset(t *testing.T) {
 			manageBuilder = manageBuilder.WithLists(&kueue.WorkloadList{Items: tc.managersWorkloads}, &jobset.JobSetList{Items: tc.managersJobSets})
 			manageBuilder = manageBuilder.WithStatusSubresource(slices.Map(tc.managersWorkloads, func(w *kueue.Workload) client.Object { return w })...)
 			manageBuilder = manageBuilder.WithStatusSubresource(slices.Map(tc.managersJobSets, func(w *jobset.JobSet) client.Object { return w })...)
-			manageBuilder = manageBuilder.WithObjects(utiltesting.MakeAdmissionCheck("ac1").ControllerName(ControllerName).Obj())
+			manageBuilder = manageBuilder.WithObjects(
+				utiltesting.MakeMultiKueueConfig("config1").Clusters("worker1").Obj(),
+				utiltesting.MakeAdmissionCheck("ac1").ControllerName(ControllerName).
+					Parameters(kueue.GroupVersion.Group, "MultiKueueConfig", "config1").
```

Comment:
```suggestion
					Parameters(kueuealpha.GroupVersion.Group, "MultiKueueConfig", "config1").
```

I understand that both kueuealpha.GroupVersion.Group and kueue.GroupVersion.Group return the same value. However, using `kueuealpha` would be proper.